common/src/main/java/com/wynntils/services/statistics/StatisticsCollectors.java

```diff
@@ -63,52 +63,60 @@ public void onMythicFoundEvent(MythicFoundEvent event) {
 
     @SubscribeEvent
     public void onRaidCompleted(RaidEndedEvent.Completed event) {
-        switch (event.getRaid()) {
-            case NEST_OF_THE_GROOTSLANGS -> {
+        switch (event.getRaid().getRaidKind().getRaidName()) {
```

Comment:
Perhaps we could put the raid kinds in static fields in RaidModel and we can compare them here without doing these string comparisons (so we don't have to update this every once in a while..)